dev-python/QtPy/QtPy-1.9.0-r3.ebuild

```diff
@@ -0,0 +1,84 @@
+	|| (
+		dev-python/PyQt5[${PYTHON_USEDEP},designer?,gles2-only?,gui?,help?,location?,multimedia?,network?,opengl?,positioning?,printsupport?,sensors?,serialport?,sql?,svg?,testlib?,webchannel?,websockets?,widgets?,x11extras?,xml(+)?,xmlpatterns?]
+		dev-python/pyside2[${PYTHON_USEDEP},designer?,gles2-only?,gui?,help?,location?,multimedia?,network?,opengl(+)?,positioning?,printsupport?,sensors?,serialport(+)?,sql?,svg?,testlib?,webchannel?,websockets?,widgets?,x11extras?,xml?,xmlpatterns?]
+	)
+
+	webengine? ( || (
+		dev-python/PyQtWebEngine[${PYTHON_USEDEP}]
+		dev-python/pyside2[${PYTHON_USEDEP},webengine]
+	) )
+
+	qml? ( || (
+		dev-python/PyQt5[${PYTHON_USEDEP},declarative]
+		dev-python/pyside2[${PYTHON_USEDEP},qml]
+	) )
+
+	quick? ( || (
+		dev-python/PyQt5[${PYTHON_USEDEP},declarative]
+		dev-python/pyside2[${PYTHON_USEDEP},quick]
+	) )
+
+	declarative? ( || (
+		dev-python/PyQt5[${PYTHON_USEDEP},declarative]
+		dev-python/pyside2[${PYTHON_USEDEP},qml,quick]
+	) )
+
+	bluetooth? ( dev-python/PyQt5[${PYTHON_USEDEP},bluetooth] )
+	dbus? ( dev-python/PyQt5[${PYTHON_USEDEP},dbus] )
+	examples? ( dev-python/PyQt5[${PYTHON_USEDEP},examples] )
+	networkauth? ( dev-python/PyQt5[${PYTHON_USEDEP},networkauth] )
+	ssl? ( dev-python/PyQt5[${PYTHON_USEDEP},ssl] )
+	webkit? ( dev-python/PyQt5[${PYTHON_USEDEP},webkit] )
+
+	3d? ( dev-python/pyside2[${PYTHON_USEDEP},3d] )
+	charts? ( dev-python/pyside2[${PYTHON_USEDEP},charts] )
+	concurrent? ( dev-python/pyside2[${PYTHON_USEDEP},concurrent] )
+	datavis? ( dev-python/pyside2[${PYTHON_USEDEP},datavis] )
+	scxml? ( dev-python/pyside2[${PYTHON_USEDEP},scxml] )
+	script? ( dev-python/pyside2[${PYTHON_USEDEP},script] )
+	scripttools? ( dev-python/pyside2[${PYTHON_USEDEP},scripttools] )
+	speech? ( dev-python/pyside2[${PYTHON_USEDEP},speech] )
+"
+
+BDEPEND="test? (
+	dev-python/PyQt5[${PYTHON_USEDEP},3d(+),bluetooth(+),charts(+),concurrent(+),datavis(+),dbus(+),declarative(+),designer(+),examples(+),gui(+),help(+),location(+),multimedia(+),network(+),networkauth(+),opengl(+),positioning(+),printsupport(+),script(+),scripttools(+),scxml(+),sensors(+),serialport(+),speech(+),sql(+),ssl(+),svg(+),testlib(+),webchannel(+),webengine(+),webkit(+),websockets(+),widgets(+),x11extras(+),xml(+),xmlpatterns(+)]
+	dev-python/PyQtWebEngine[${PYTHON_USEDEP}]
+	dev-python/pyside2[${PYTHON_USEDEP},3d(+),bluetooth(+),charts(+),concurrent(+),datavis(+),dbus(+),designer(+),examples(+),gui(+),help(+),location(+),multimedia(+),network(+),opengl(+),positioning(+),printsupport(+),qml(+),quick(+),script(+),scripttools(+),scxml(+),sensors(+),serialport(+),speech(+),sql(+),svg(+),testlib(+),webchannel(+),webengine(+),webkit(+),websockets(+),widgets(+),x11extras(+),xml(+),xmlpatterns(+)]
+)"
+
+distutils_enable_tests pytest
+
+python_test() {
+	export QT_API="pyqt5"
+	virtx pytest -vv || die "Tests failed with ${EPYTHON} and ${QT_API}"
+	export QT_API="pyside2"
+	virtx pytest -vv || die "Tests failed with ${EPYTHON} and ${QT_API}"
```

Comment:
We are advised to not put `|| die` in `virtx` commands, 
https://devmanual.gentoo.org/eclass-reference/virtualx.eclass/index.html

Reply:
Oopsie, I forgot about that. Fixed it now, thanks